pkg/apis/config/validation/account.go

```diff
@@ -94,3 +111,13 @@ func ValidateRoleBindingSubject(subject rbac.Subject, isNamespaced bool, fldPath
 
 	return allErrs
 }
+
+func ValidateAccountSpaceTemplate(space configv1alpha1.AccountSpaceTemplate, fldPath *field.Path) field.ErrorList {
```

Comment:
Currently this function checks if the AccountSpaceTemplate (`spec.space.spaceTemplate`) is empty and returns an error if it is. I'm sorry if this was a little bit misleading in the issue description, but the function should check instead if the label `kiosk.sh/account` (constants.SpaceLabelAccount) in `spec.space.spaceTemplate.metadata.labels` is empty and if it is NOT, then return an invalid error. All other metadata in spaceTemplate is fine and is allowed.

The problem with this label is that we set it on the namespace and is essentially the identifier which account "owns" this namespace. Hence this label is not allowed in a spaceTemplate that will be used during space creation, because a user could essentially override the `Space.spec.account` property with that label. Hope this clears up the problem.